dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -35,6 +36,62 @@ export const JobDetailChartsPage = ({
   const { job, msg, params } = useJobDetail();
   const jobId = params.id;
 
+  const { cluster_status } = useRayStatus();
+  console.log(cluster_status?.data.clusterStatus.split("\n"));
+
+  const FormatNodeStatus = (cluster_status: string) => {
+    // ==== auto scaling status
+    // Node status
+    // ....
+    // Resources
+    // ....
+    const sections = cluster_status.split("Resources");
+    return FormatClusterStatus(
+      "Node Status",
+      sections[0].split("Node status")[1],
+    );
+  };
+
+  const FormatResourcesStatus = (cluster_status: string) => {
+    // ==== auto scaling status
+    // Node status
+    // ....
+    // Resources
+    // ....
+    const sections = cluster_status.split("Resources");
+    return FormatClusterStatus("Resource Status", sections[1]);
```

Comment:
```suggestion
    return formatClusterStatus("Resource Status", sections[1]);
```

Reply:
so inlined component should start with lowercase?